datalad/crawler/nodes/tests/test_annex.py

```diff
@@ -24,6 +24,21 @@
 from ....consts import CRAWLER_META_CONFIG_PATH, DATALAD_SPECIAL_REMOTE, ARCHIVES_SPECIAL_REMOTE
 from ....support.stats import ActivityStats
 from ....support.annexrepo import AnnexRepo
+from os import listdir
+
+
+@with_tempfile(mkdir=True)
+def test_Annexificator(outdir):
+
+    eq_(listdir(outdir), [])     # directory is new and empty
+
+    filed = open(outdir+'/myfile.txt', 'w+')
```

Comment:
use `os.path.join` which we usually import as `opj`.

generally adviced pattern is to use 'with' statement to operate on the files